jsapp/js/components/modalForms/encryptForm.es6

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import autoBind from 'react-autobind';
+import {bem} from 'js/bem';
+import TextBox from 'js/components/textBox';
+import {t} from 'utils';
+import {actions} from 'js/actions';
+
+class EncryptForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      submissionURL: '',
+      publicKey: '',
```

Comment:
You should read current `submissionURL` and `publicKey` values, and load them in the modal.